scripts/install.sh

```diff
@@ -22,7 +30,6 @@ DIR="${BASH_SOURCE}"
 if [ ! -d "$DIR" ]; then DIR="$PWD/scripts"; fi
 . "$DIR/require.sh"
 
-ENV=$1
```

Comment:
What if `$ENV` is not defined in the loop?

Reply:
same as if I do `echo $NOTDEFINEDVAR`, just get none/empty string (does bash have the concept of undefined var?). Or same as before if I execute the script with no arguments.

Reply:
Bash is permissive by default when dealing with undefined vars. You can set an option to enable a restrictive mode which will fail the whole script if it encounters a single undefined var but it's not the case here.
Anyway, this PR does not change the former behavior regarding `$ENV` so I don't have any objection.
You can add a new commit or open a new PR to handle the specific case of empty `$ENV` :)